### PR TITLE
More repo details & Skip welcome page

### PR DESCRIPTION
**Pull request by tyn1998**
# More repo details
image

# Skip welcome page
image

### CODE REVIEW

**Comment by frank-zsy** [comment]
I will close and reopen this PR for CI test.

File: src/SODAComponents/GitHubScreen/GitHubScreen.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import ContributorsGeoDistribution from "./ContributorsGeoDistribution/Contribut
 import CompaniesScatter from "./CompaniesScatter/CompaniesScatter.jsx";
 
 // var BASE_URL = "http://192.168.3.16:8081";
-var BASE_URL = "http://nzcer.cn:8081";
+var BASE_URL = "https://nzcer.cn:8081";
```
Comment (frank-zsy) [inline src/SODAComponents/GitHubScreen/GitHubScreen.jsx:16]: What is this URL used for? I got errors while visiting it.

**Comment by frank-zsy** [comment]
LGTM

/approve

File: src/SODAComponents/GitHubScreen/GitHubScreen.jsx (hunk 1 above)
Comment (tyn1998) [reply src/SODAComponents/GitHubScreen/GitHubScreen.jsx:16]: This is for the Data Screen toggled by ` key. The data now is provided by a java service, I think it should be stored as files on oss. I'll work on this.

Comment (frank-zsy) [reply src/SODAComponents/GitHubScreen/GitHubScreen.jsx:16]: Yes, it should be on OSS too, I can help with this too later.

**Pull request merged**